src/components/SideNav/SideNav.tsx

```diff
@@ -26,6 +26,28 @@ const SideNav = (props: SideNavProps): ReactElement => {
 		link.onClick();
 	};
 
+	const findLinkselected = () => {
+		let linkSelected: Link
+		links.forEach(link => link.forEach(l => l.label === defaultLink ? linkSelected = l : null))
+		if (!linkSelected) {
+			throw new Error("Default link is not in the links array.");
+		} else {
+			return linkSelected
+		}
+	}
+
+	useEffect(() => {
+		if  (links) {
+			if (defaultLink) {
+				setSelectedLink(defaultLink)
+				onLinkClicked(findLinkselected())
+			} else {
+				setSelectedLink(links[0][0].label)
+				onLinkClicked(links[0][0])
+			}
+		}
+	}, [defaultLink])
```

Comment:
This will most probably throw a dependency warning because we should also be including "links" here since it's being used inside the useEffect.
```suggestion
	}, [links, active])
```